BondageClub/Screens/Room/Nursery/Nursery.js

```diff
@@ -58,7 +58,7 @@ function NurseryLoad() {
 			NurseryNurse.Stage = "270";
 			NurseryLoadNurse();
 		} else {
-			if (CharacterAppearanceGetCurrentValue(Player, "Panties", "Name") != "Diapers1") {
+			if (CharacterAppearanceGetCurrentValue(Player, "Panties", "Name").toUpperCase().indexOf("DIAPER", 0) < 0) {
```

Comment:
Suggestion for better code readability:
```suggestion
			if (!CharacterAppearanceGetCurrentValue(Player, "Panties", "Name").toUpperCase().includes("DIAPER", 0)) {
```